### ISSUE TITLE
1.19.2 Forge support

### ISSUE THREAD

**Issue by EBLeifEricson** [IsaiahMC/multiworld#43]
While there are 1.19.2/1.19.3 builds for Fabric, only a 1.19 Forge build currently exists. It would be helpful to also have an equivalent Forge build for 1.19.2+.

**Comment by Zalatis** [comment]
Would really like this too !

**Comment by Andarius68** [comment]
Agreed. I did try to load the 1.19 Forge build with my 1.19.4 Forge client but it crashes on loading. This was both with the 1.5 Release build and the 1.6. Ordinarily I wouldn't mind rolling back to 1.19 in order to use the mod but I was hoping to use it in conjunction with the new release of the Aether mod which, unfortunately, only supports 1.19.4 currently.

**Comment by BrodyDurand** [comment]
I would also love for a 1.19.2 forge version. I thought that it was strange that fabric has a version but forge does not. It would be extremely helpful for my server for my friends for world management.

**Comment by Thalia-the-nerd** [comment]
where can i find this version for 1.19.2 forge as its been marked as compleated
